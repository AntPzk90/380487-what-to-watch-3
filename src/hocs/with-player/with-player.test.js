import React from 'react';
import renderer from 'react-test-renderer';
import withPlayer from './with-player.jsx';


it(`SnapshotTest withPlayer`, () => {

  const tree = renderer
  .create(
      <withPlayer
      isElepsed={0}
      isPlaying={true}
      isPause={false}
      fullTime={null}
      isFullScreen={false}
      onFullScreenCkick={() => {}}
      onPlayBtnClick={() => {}}
      playerRef={() => {}}
      onExitBtnClick={() => {}}
      />
  )
  .toJSON();

  expect(tree).toMatchSnapshot();
});
