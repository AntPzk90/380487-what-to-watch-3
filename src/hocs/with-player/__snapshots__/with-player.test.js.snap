// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapshotTest withPlayer 1`] = `
<withPlayer
  fullTime={null}
  isElepsed={0}
  isFullScreen={false}
  isPause={false}
  isPlaying={true}
  onExitBtnClick={[Function]}
  onFullScreenCkick={[Function]}
  onPlayBtnClick={[Function]}
  playerRef={[Function]}
/>
`;
